webcam.go

```diff
@@ -35,7 +35,7 @@ type Control struct {
 // Checks if device is a v4l2 device and if it is
 // capable to stream video
 func Open(path string) (*Webcam, error) {
-	handle, err := unix.Open(path, unix.O_RDONLY|unix.O_NONBLOCK, 0666)
+	handle, err := unix.Open(path, unix.O_RDWR|unix.O_NONBLOCK, 0666)
```

Comment:
I see documentation for https://github.com/blackjack/webcam/pull/69/files#diff-f8a6427913c9b96c74a5890121c6b1a57065eae1084267f2935352359b3ab27dR502 in v4l2 docs you linked to: https://www.kernel.org/doc/html/v4.9/media/uapi/v4l/func-mmap.html but I don't see anything about this change. Was this `Open` failing for you? If so, do you know why?